added download information (#1)
WayScience/mitocheck_movies: README.md

# mitocheck_movies

> All information regarding the download and processing of Mitocheck whole movies from IDR study with accession idr0013 (screenA). Movies are downloaded with Aspera. 

Reviewer: If available, consider providing a link to `idr0013 (screenA)` for more details.

Author: linked!

## Aspera Downloader:

### Step 1: Install Aspera

> Install Aspera high-speed transfer client as described at https://github.com/IBM/aspera-cli#installation. We used the direct installation method. On Ubuntu, the direct installation method is as follows:

Reviewer: Consider using one-sentence-per-line documentation style with this line and throughout the document. Using this style helps with revising, reviewing, and tracking changes to the content.

Author: changed.

> Install Aspera high-speed transfer client as described at https://github.com/IBM/aspera-cli#installation. We used the direct installation method. On Ubuntu, the direct installation method is as follows:

Reviewer: Consider wrapping the URL here as a markdown link (for ex `[link text](url)`).

Author: created hyperlinks

1. Install Ruby on Ubuntu: `sudo apt install ruby-full`
2. Confirm Ruby install: `ruby --version`
3. Install aspera-cli gem: `gem install aspera-cli`
4. Upgrade aspera-cli gem to latest version: `gem update aspera-cli`
5. Install acsp: `ascli conf ascp install`

> 6. Confirm install and locate acsp (for step 1a): `ascli config ascp show`

Reviewer: Very clear documentation! Would it make sense to move these steps into an installation script of some kind?

### Step 1a: Allow Aspera to run without password

> It is necessary to allow Aspera command line interface (ascp) to run without a password to use it more easily with python. In order to run ascp without a password do the following:

Reviewer: Consider providing a background on which password (system, Aspera, etc) is required and why the steps need to be taken. Additionally, seeing the `sudo` inclusion below makes me wonder if file permissions work might be addressed through a reference to the [ascp CLI docs: Required File Access and Permissions](https://www.ibm.com/docs/en/aci/3.9.2?topic=ascp-command-reference#d8444e314) (I don't know for sure, and only make this reference in case it helps). For example, modifying sudo config or running sudo without a password may not be possible or advisable on some systems.

1. Type `sudo visudo` at the terminal to open the sudo permissions (sudoers) file
2. Find the line with: `%sudo   ALL=(ALL:ALL) ALL`
3. Below that line, insert the following line: `username  ALL=(ALL) NOPASSWD: path_to_aspera`
Note: `username` is your username and `path_to_aspera` can be found with `ascli config ascp show`
4. Save and exit the editor

### Step 2: Download Aspera Public Key

> Download and Aspera public key. You will need the location of this key to use Aspera

Reviewer: If available, consider providing a link to documentation covering where one might find the public key for Aspera.

# mitocheck_movies
All information regarding the download and processing of Mitocheck whole movies from IDR study with accession idr0013 (screenA) ([paper](https://pubmed.ncbi.nlm.nih.gov/20360735/idr), [data](https://idr.openmicroscopy.org/webclient/?show=screen-1101), [metadata](https://github.com/IDR/idr-metadata/blob/master/idr0013-neumann-mitocheck/screenA/)). 
Movies are downloaded with Aspera using `download.py`. 
Note that the full dataset is > 20 TB so the number of plates to download should be specified by changing the `plate_subset` variable.
Then illumination correction is performed with PyBaSiC and imageJ using either `process_single_movie.py` or `process_multiple_plates.py`. 

> The number of frames to include in the shortened video can be changed with the `frames` variable and the image compression can be controlled using the `down_factor` variable. 

Reviewer 2: ```suggestion
You can change the number of frames to include in the shortened video using the `frames` variable and you can control the image compression using the `down_factor` variable. 
```

Switch to active voice. Also, I love these notes! They should also be in the .py file (and possible notebook) as well.

### Step 2: Download Aspera Public Key
Download and Aspera public key. 
You will need the location of this key to use Aspera

## Image Processing  

### Step 1: Install Fiji

Download the version of [Fiji](https://imagej.net/software/fiji/) (ImageJ) compatible with your system. 
Fiji is used to open images.

### Step 2: Install PyImageJ

 [PyImageJ](https://github.com/imagej/pyimagej)is the python wrapper function for ImageJ.

>  ### Step 3: Install PyBaSic

Reviewer 2: ```suggestion
### Step 3: Install PyBaSic
```

typo
